tests/commerce_integration/TEC/Tickets/Commerce/Stock/SharedCapacity/SharedCapacityTest.php

```diff
@@ -441,4 +441,90 @@ public function test_tc_total_shared_capacity_decrease_should_decrease_capped_ca
 
 		$this->assertEqualSets( $expected_ticket_counts, $event_ticket_counts['tickets'] );
 	}
+
+	public function test_tc_total_shared_capacity_should_not_change_after_unlimited_capacity_ticket_purchase() {
+		$event_id = tribe_events()->set_args(
+			[
+				'title'      => 'Test Event',
+				'status'     => 'publish',
+				'start_date' => '2020-01-01 12:00:00',
+				'duration'   => 2 * HOUR_IN_SECONDS,
+			]
+		)->create()->ID;
+		// Enable the global stock on the Event.
+		update_post_meta( $event_id, Global_Stock::GLOBAL_STOCK_ENABLED, 1 );
```

Comment:
Is there a reason we can't use `Tribe__Tickets__Global_Stock::enable()` here?

Reply:
Not really. The only difference between this and using the `enable()` method would be two lines of code vs one to accomplish the same result. For testing purposes, this is all we need.